include/vnx/addons/TcpServer.h

```diff
@@ -95,7 +100,7 @@ class TcpServer : public TcpServerBase {
 private:
 	int m_socket = -1;
 #ifdef _WIN32
-	int m_notify_socket = -1;
+	SOCKET m_notify_socket = INVALID_SOCKET;
```

Comment:
why not the same for `HttpServer.h` ?

Reply:
Looks like this change not needed, reverted